nemo/collections/asr/modules/rnnt_abstract.py

```diff
@@ -285,6 +285,13 @@ def batch_replace_states_mask(
         """Replace states in dst_states with states from src_states using the mask, in a way that does not synchronize with the CPU"""
         raise NotImplementedError()
 
+    @classmethod
+    def batch_replace_states_all(
```

Comment:
There is nothing related to the `self`, so I prefer `classmethod` for this case. Also, I would prefer consistency with `batch_replace_states_mask`, which is also a `classmethod`.
For reusability purposes, `classmethod` is also better since if we know that this state is related to some class, we can use `classmethod` without requiring us to pass the actual decoder instance.